### ISSUE TITLE
pdfsizeopt destroys Azure invoice (blank pages on output)

### ISSUE THREAD

**Issue by hannob** [pts/pdfsizeopt#117]
When I run pdfsizeopt on an invoice I got from Microsoft Azure I get a broken document, the output only contains white pages.

I don't want to upload my invoice containing payment and account data here, but I'd share it with you privately via email if you want a sample.

**Comment by pts** [comment]
Thank you, please send me a sample PDF. You can find my e-mail address here: https://github.com/pts/pdfsizeopt/blob/5a42db5bef4df1652adb3ef49a9d14bcb58442eb/pdfsizeopt_test.py#L21

Please also copy-paste the console output pdfsizeopt to this GitHub issue, so that I can compare it with the output I get. (The console output doesn't contain any secrets.)

**Comment by hannob** [comment]
pdfsizeopt output:
```
info: This is pdfsizeopt ZIP rUNKNOWN size=69649.
info: prepending to PATH: /data/software/pdfsizeopt/pdfsizeopt_libexec
info: loading PDF from: 2019-05-21-azure.pdf
info: loaded PDF of 120117 bytes
info: separated to 30 objs + xref + trailer
warning: incorrect /Length fixed for obj 7
warning: incorrect /Length fixed for obj 18
warning: incorrect /Length fixed for obj 20
warning: incorrect /Length fixed for obj 22
warning: incorrect /Length fixed for obj 24
info: parsed 30 objs
info: found 0 Type1 fonts loaded
info: found 0 Type1C fonts loaded
warning: error decompressing obj 7: Flate decompression error: Error -3 while decompressing: incorrect header check
warning: error decompressing obj 18: Flate decompression error: Error -3 while decompressing: incorrect header check
warning: error decompressing obj 20: Flate decompression error: Error -3 while decompressing: incorrect header check
warning: error decompressing obj 22: Flate decompression error: Error -3 while decompressing: incorrect header check
warning: error decompressing obj 24: Flate decompression error: Error -3 while decompressing: incorrect header check
info: optimized 7 streams, kept 5 #dec-error, 2 zip
info: eliminated 1 duplicate objs
info: compressed 0 streams, kept 0 of them uncompressed
info: saving PDF with 29 objs to: out.pdf
info: generated object stream of 1769 bytes in 21 objects (16%)
info: generated 108719 bytes (91%)

**Comment by pts** [comment]
Thank you for reporting this! Fixed in 33ec5e5c637fc8967d6d238dfdaf8c55605efe83. Download the newest pdfsizeopt, and try again.

The PDF reference says that `stream` must be either `'stream\n'` or `'stream\r\n'`. The input file *2019-05-21-azure.pdf* is thus invalid, because it has `'stream \r\n'' (i.e. it has an extra space between `stream` and the CRLF). I've made pdfsizeopt more permissive to accept this, by making it ignore the extra space.
